packages/node/Dockerfile

```diff
@@ -35,6 +35,9 @@ RUN rm /app.tgz && \
     yarn cache clean && \
     rm -rf /root/.npm /root/.cache
 
+# Make the user not ROOT
+USER 1000
```

Comment:
I think this is not enough. You need to create the user in the container, and chown so the subql sdk binaries can be used by user 1000

Reply:
I've tested this and things work as expected